File: Week1_C/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int coin[4];

    coin[0] = 25;
    coin[1] = 10;
    coin[2] = 5;
    coin[3] = 1;

    int cents = 0;

    do {
        cents = get_int("Change owed: ");
    } while (cents < 0);

    int total = 0;
    int index = 0;

    while (cents > 0){
        if (cents - coin[index] >= 0 ) {
            cents -= coin[index];
            total++;
        }
        else index++;
    }

    printf("%d\n", total);

    return 0;
}

File: Week6_Python/sentimental-cash/cash.py
coin = [25, 10, 5, 1]

value = 0;
while value <= 0:
    try:
        value = float(input("Change: "))*100
    except:
        print("Erro")

total = 0;
index = 0;

while value > 0:
    if (value - coin[index]) >= 0:
        value -= coin[index]
        total += 1
    else:
        index += 1

print(total)

